articles/Getting-started-with-windows-forms-using-c-#/index.md

```diff
@@ -0,0 +1,176 @@
+**Windows forms** allow the programmers to create a graphical user interface (GUI) that allows the interaction with the program visually. According to [Microsoft](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/overview/?view=netdesktop-5.0), It provides one of the most productive ways to create desktop apps based on the visual designer provided in Visual Studio. Functionality such as drag-and-drop placement of visual controls makes it easy to build desktop apps.
+
+With *windows forms*, you can make basic applications like a calculator to high-level applications such as a library database management application that requires a connection to the database from the application.
+### Prerequisites
+1. A basic understanding of c# programming language.
+2. Visual studio installed on your system.
+
+If you don't have Visual Studio installed on your computer. you can download it from [Here](https://visualstudio.microsoft.com/downloads/).
+
+### Make a new project
+In this tutorial, we will make a simple calculator! Make a new project by opening the visual studio, then click on File -> New -> Project -> Visual c# -> Windows desktop and choose Windows Forms App (.NET Framework).
+
+### ToolBox
+The *Toolbox* window provides you with all the controls that help you build your project inside the form. to display the Toolbox click on View -> Toolbox, or you can press Ctrl+Alt+X. To use one of the controls double click on one of them or simply drag and drop it.
+
+### Properties window
+The *Properties window* is used to show the properties of the form or the control that you click on, you can manipulate the properties of any control used in your form. To display the window you can click on View -> properties window or you can simply click F4.   
```

Comment:
The *Properties window* is used to show the properties of the form or the control that you click on,

Instead of a comma, break the sentence with a period (`.`)

Reply:
Instead of `To display the window...`, be specific and say `To open the properties window, ...`

Don't forget to add a comma.

Reply:
Make sure you capitalize on all instances after `->`